server/src/middleware/merchant-auth.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Request, Response, NextFunction} from 'express';
+import admin from 'firebase-admin';
+
+admin.initializeApp();
```

Comment:
Should we do something similar to the client side, creating a `src/firebase.ts` which initializes the app and exports the `const auth = admin.auth();`?

Reply:
Yep, I think I've already done that  https://github.com/googleinterns/gpay-group-buy/pull/54/commits/2f1b38df7a3993c6eb523b4b94badb447d99ecd2

Reply:
@karenfrilya97 Isn't that on the client side? I actually meant like doing the same for server side too